src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ciagmigration/jpa/entity/induction/ParentMigrationEntity.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration.jpa.entity.induction
+
+/**
+ * Abstract class for "parent" entities, which contain one or more collections of [KeyAwareChildMigrationEntity] entities.
+ */
+abstract class ParentMigrationEntity {
+
```

Comment:
This is the same as the existing `ParentEntity`, except it doesn't have the method:

```
abstract fun childEntityUpdated()
```

This is because we are only going to use these entities for inserts, not updates, and we will use the timestamps that we import from the CIAG API.